components/ProductDetails/ProductDetailsItinerary.vue

```diff
@@ -80,6 +80,10 @@ h2, h3, .expand {
     color: #103A5B;
 }
 
+#itinerary h3 {
+    font-size: 14px;
```

Comment:
Please check if we need responsive font size
https://www.w3schools.com/howto/howto_css_responsive_text.asp

Reply:
I will check with the designers.